tests/court_tests/test_court.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+from court.models import Court
+
+
+TEST_ID = 1
+TEST_X = 1.234567
+TEST_Y = 9.876543
+TEST_CITY = 'New York'
+TEST_NEIGHBORHOOD = 'Hogwarts'
+TEST_MAX_PLAYERS = 5
+
+
+@pytest.fixture
+def court():
+    return Court(id=TEST_ID, x=TEST_X, y=TEST_Y, city=TEST_CITY, neighborhood=TEST_NEIGHBORHOOD,
+                 max_players=TEST_MAX_PLAYERS)
+
+
+@pytest.mark.django_db
+class TestCourtModel:
+
+    def test_court_model_save_method(self):
+        court = Court.objects.create(
+            id=TEST_ID,
+            x=TEST_X,
+            y=TEST_Y,
+            city=TEST_CITY,
+            neighborhood=TEST_NEIGHBORHOOD,
+            max_players=TEST_MAX_PLAYERS
+        )
+        assert court.id == TEST_ID
+        assert court.x == TEST_X
+        assert court.y == TEST_Y
+        assert court.city == TEST_CITY
+        assert court.neighborhood == TEST_NEIGHBORHOOD
+        assert court.max_players == TEST_MAX_PLAYERS
```

Comment:
I think it will make more sense to fetch the instance from the db and then compare its fields to the object in the fixture.